lib/ASTGen/Sources/ASTGen/ParameterClause.swift

```diff
@@ -121,11 +147,32 @@ extension ASTGenVisitor {
       rightParenLoc: self.generateSourceLoc(node.rightParen)
     )
   }
+
+  func generate(accessorParameters node: AccessorParametersSyntax) -> BridgedParameterList {
+    let (name, nameLoc) = self.generateIdentifierAndSourceLoc(node.name)
+    let param = BridgedParamDecl.createParsed(
+      self.ctx,
+      declContext: self.declContext,
+      specifierLoc: nil,
+      argName: nil,
+      argNameLoc: nil,
+      paramName: name,
+      paramNameLoc: nameLoc,
+      type: nil,
+      defaultValue: nil
+    )
+    return .createParsed(
+      self.ctx,
+      leftParenLoc: self.generateSourceLoc(node.leftParen),
+      parameters: CollectionOfOne(param).bridgedArray(in: self),
+      rightParenLoc: self.generateSourceLoc(node.rightParen)
+    )
+  }
 }
 
 extension ASTGenVisitor {
   @inline(__always)
-  func generate(functionParameterList node: FunctionParameterListSyntax) -> BridgedArrayRef {
-    node.lazy.map(self.generate).bridgedArray(in: self)
+  func generate(functionParameterList node: FunctionParameterListSyntax, forSubscript: Bool) -> BridgedArrayRef {
+    node.lazy.map { self.generate(functionParameter: $0, forSubscript: forSubscript) }.bridgedArray(in: self)
```

Comment:
Just a style preference, for method chaining like this, I'd prefer enclosing the closure in parentheses `map({ ... }).bridgedArray`